modules/userId/index.js

```diff
@@ -1211,6 +1223,10 @@ export function init(config, {delay = GreedyPromise.timeout} = {}) {
         initIdSystem({ready: true});
       }
     }
+    if (!addedUserIdHook && !startAuction.getHooks({hook: addUserIdsHook}).length) {
```

Comment:
I'd put it outside of the config listener - like [this](https://github.com/prebid/Prebid.js/commit/4ccf40e0b13976eb3b89baaaa7be0c46035a234f). Do you have a reason to prefer it here? I'm thinking that without any submodules, you wouldn't get any configuration either.